src/applications/personalization/dashboard-2/tests/unit/appointments.unit.spec.jsx

```diff
@@ -0,0 +1,91 @@
+// Ensure sorting is done correctly (startsAt wise)
```

Comment:
This testing coverage is very basic and we're going to need to add more specific unit tests for used helpers (i.e. timezone logic)